enums.py

```diff
@@ -146,6 +156,34 @@ def to_java_upper(symbol):
         f.write('\n')
     f.write('YG_EXTERN_C_END\n')
 
+# write out C headers for printing
+with open(root + '/yoga/YGEnumsPrint.h', 'w') as f:
```

Comment:
Could this just be part of YGEnums.h? Collocating this stuff is usually pretty nice